lib/connection_pool.rb

```diff
@@ -32,7 +33,7 @@
 # - :timeout - amount of time to wait for a connection if none currently available, defaults to 5 seconds
 #
 class ConnectionPool
-  DEFAULTS = {size: 5, timeout: 5}
+  DEFAULTS = {size: 5, timeout: 5, reaping_frequency: nil, reap_after: 60}
```

Comment:
the choice to make `reaping_frequency` the trigger to turn on reaping is to make it behave like ActiveRecord.

60s is a short `reap_after` in my opinion though.